### PR TITLE
Update pretalx2wtd to retrieve speaker info and other cleanup.

### PR DESCRIPTION
**Pull request by mxsasha**
The script now does its own API calls, and has a bit more
logging. Twitter and website are added to the sessions file
for Portland 2020. Order has changed.

### CODE REVIEW

**Comment by mxsasha** [comment]
Nobody seems to have an avatar now, so we can add that feature once some speakers have uploaded one. Shouldn't be too hard now that we're retrieving full speaker info already for each speaker.

**Comment by plaindocs** [comment]
This looks awesome. 

At least one speaker does have an Avatar, Lucy Cary.
They are also commented out on the speaker page till we have more.

**Comment by mxsasha** [comment]
Oh, actually it's already in there. Do we want to make our own download, or just defer to pretalx? I don't like to manage automatic downloads, so I'm leaning to just do it like this:

```
  speakers:
  - name: Lucy Carey
    slug: lucy-carey
    avatar: https://pretalx.com/media/avatars/Screenshot_2020-02-19_at_09.56.26_pUIrHqc.png
    twitter: lucyrushi
```

**Comment by plaindocs** [comment]
I'd be tempted to run it manually and update the git repo with the images so we're not dependent on Pretalx at build time. 

So wget the image and save to > images/slug.img 

What do you think @mxsasha

**Comment by mxsasha** [comment]
> I'd be tempted to run it manually and update the git repo with the images so we're not dependent on Pretalx at build time.

I'll add that.

**Comment by mxsasha** [comment]
I've added downloading of speaker images. Note that the location is derived from the slug in the speaker_photo template filter, so we don't actually need the avatar location in the sessions YAML, so I removed that. The speaker_photo template filter should be compatible with this.

**Comment by mxsasha** [comment]
@plaindocs I think this is now ready for merging, can you approve the PR if you agree?

File: docs/_scripts/pretalx2wtd.py
Comment (plaindocs) [inline docs/_scripts/pretalx2wtd.py:32]: `{pretalx_slug}` needs to available here. Not sure if you use any of the other

Comment (mxsasha) [reply docs/_scripts/pretalx2wtd.py:32]: I am confused. It is available. The script runs fine?

**Comment by mxsasha** [comment]
@plaindocs I added the current output for the Portland 2020 sessions in this PR too. The script seems to run fine, and I think it was all done.

So I think we can merge this, which will also update the Portland 2020 sessions - you should probably have a look over https://github.com/writethedocs/www/blob/pretalx-improvements/docs/_data/portland-2020-sessions.yaml to see if it seems correct.

**Review by plaindocs — APPROVED** [review]
This is awesome!

**Pull request merged**
